server.js

```diff
@@ -0,0 +1,118 @@
+const express = require('express');
+const db = require('./data/db');
+
+const port = 5555;
+const server = express();
+server.use(express.json());
+
+server.get('/', (req, res) => {
+	//1st arg: route where a resource can be interacted with
+	//2nd arg: callback to deal with sending responses, and handling incoming
+  res.send('Hello from express');
+})
+
+server.post('/api/users', (req, res) => {
+	if (!req.body.name || !req.body.bio) {
+		res.status(400);
+		res.json({ errorMessage: "Please provide name and bio for the user." });
+	}
```

Comment:
Out of curiosity - what made you decide to use req.body.name, etc instead of creating a variable `const { name, bio } = req.body;` and then just `if(!name || !bio)`? Both work, I'm just curious why you did it this way :)

Reply:
Oh look, you did use the const _right below this_. Now I'm double curious since you used both in the same code block. Are you just playing around and experimenting with different ways?